backend/account/migrations/0002_auto_20230718_1040.py

```diff
@@ -1,30 +1,31 @@
 # mypy: ignore-errors
 # Generated by Django 4.2.1 on 2023-07-18 10:40
 
+from django.conf import settings
 from django.contrib.auth.hashers import make_password
 from django.db import migrations
 
 
 def create_public_tenant_and_domain(apps, schema_editor):
-    Organization = apps.get_model("account", "Organization")
+    organization_model = apps.get_model("account", "Organization")
```

Comment:
NIT: @muhammad-ali-e why have we renamed it here? We don't seem to follow this for other data migrations

Reply:
Sonar issue raised : variable name  should follow the snakecase

Reply:
Ah got it, maybe we could try importing the models within this function. That way we can probably type it too. We might need `from __future__ import annotations` alone